content/docs/1.0.x/manage/debug/index.md

```diff
@@ -9,14 +9,20 @@ aliases:
 Modifying the log level preserves more detailed information in the logs.
 This can be useful when analyzing execution problems for your project.
 
-To do this:
+You can set the log level in either of two ways:
 
-* Adjust the value of the `LOG_LEVEL` environment variable
-  in the shipyard controller to `debug`.
-  Alternately, you can adjust the `loglevel` Helm value to `debug`.
+* Adjust the `loglevel` value to `debug`
+  in the [Helm chart](../../reference/files/values)
+  or by using the [-set flag](../../install/helm-install/#the---set-flag)
+  on the **helm** command line.
```

Comment:
Using `--set` to set values is generally not the best practise way of setting value, I suggest not advertise it. Generally, we have docs on how to set helm values anyways, I don't think we need to reiterate them here.
```suggestion
  in the [Helm chart](../../reference/files/values).
```

Reply:
Not reiterating but xref'ing to the other info.  But it's gone.